Homogeneous_kernel/include/CGAL/Homogeneous/function_objects.h

```diff
@@ -3098,12 +3098,12 @@ namespace HomogeneousKernelFunctors {
     }
 
     Point_2
-    operator()(const Line_2& l, int i) const
+    operator()(const Line_2& l, const FT i) const
```

Comment:
Is not it an issue to rely on `FT` in the homogeneous kernel?

Reply:
It could be... Could you please precise what you mean? In my head, `FT` was just `Quotient<template_number_type>` so here the user gives me some quotient, and I do some vector scaling? I guess I might change the `w` in the homogeneous point in output, but it doesn't seem like an issue?

Reply:
I think there should be an overload for `int` or `RT` as well, so that the code involved is not complicated by the `FT` case. It seems one can use `typename First_if_different<FT_,RT>::Type` to have SFINAE one of the two overloads when `FT` and `RT` are equal.

Reply:
Done in a45ccab. Can't do anything for other types because impressingly `ray_23/line_23.point(i)` all manage to take different approaches to achieve the same construction. Probably could do some kernel-wide uniformizing!

Reply:
Needed some more work because there is some code using `FT := int` (which doesn't seem very legal, `int` is not a model of `FieldNumberType`...?)